tooling/Microsoft.MonoDevelop.RazorAddin/Microsoft.MonoDevelop.RazorAddin.csproj

```diff
@@ -0,0 +1,72 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="AddinMetadata.props" />
+  
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <EnableApiCheck>false</EnableApiCheck>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Properties\_Manifest.addin.xml" LogicalName="_Manifest.addin.xml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- 
+      By default the MonoDevelop.Addins package has restore tasks that depend on monodevelop existing on the machine. 
+      We can avoid this requirement by not letting it auto-restore monodevelop dependencies because we know which ones
+      we depend on and can bring them in manually.
+    -->
+    <PackageReference Include="MonoDevelop.Addins" Version="$(MonoDevelopAddinsVersion)" NoWarn="KRB4002" ExcludeAssets="build" />
+    <Reference Include="Mono.Addins">
+      <HintPath>$(NuGetPackageRoot)monodevelop.addins\$(MonoDevelopAddinsVersion)\build\Mono.Addins.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- 
+      The extension project can not have a direct reference to its language service pieces. They are included via the manifest above.
+      This piece ensures that transitive builds/restores occurr for this project.
+    -->
+    <ProjectReference Include="..\..\src\Microsoft.MonoDevelop.LanguageServices.Razor\Microsoft.MonoDevelop.LanguageServices.Razor.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <OutputItemType>Content</OutputItemType>
+      <Targets>Build</Targets>
+    </ProjectReference>
+  </ItemGroup>
+
+  <!-- We need to generate assembly attributes with build time information to let MonoDevelop know RazorAddins metadata at runtime. -->
+  <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+    <PropertyGroup>
+      <GeneratedAddinAssemblyInfo>$(IntermediateOutputPath)$(MSBuildProjectFile).AddinInfo.cs</GeneratedAddinAssemblyInfo>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <AssemblyAttributes Include="Mono.Addins.AddinAttribute">
+        <_Parameter1>$(AddinId)</_Parameter1>
+        <Namespace>$(AddinNamespace)</Namespace>
+        <Version>$(AddinVersion)</Version>
+      </AssemblyAttributes>
+      <AssemblyAttributes Include="Mono.Addins.AddinNameAttribute">
```

Comment:
Maybe call this `RazorAssemblyAttribute`? It's kinda close to `AssemblyAttribute` which is the name the SDK uses for it's itemgroup so might be confusing.